static/src/components/PlayerDisplay.js

```diff
@@ -62,113 +57,205 @@ function mapDispatchToProps(dispatch) {
 
 export class PlayerDisplay extends Component {
 
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
+        console.log("constructor");
```

Comment:
Don't need this console.log

Reply:
This was just me checking when the constructor was accessed. I will delete :)